tests/population/test_adapter.py

```
@@ -36,13 +38,14 @@ def simulate(self, n_samples: int, **kwargs):
 class MockExternalPopulationLoader:
     """Protocol-compatible file loader backend for adapter tests."""
 
-    parameter_names = (
+    parameter_names: ClassVar[tuple[str, ...]] = (
         "detector_frame_mass_1",
         "detector_frame_mass_2",
         "inclination",
         "coa_time",
     )
     source_type = "bns"
+    metadata: ClassVar[dict[str, str]] = {"resolved_path": "/tmp/catalog.h5"}
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/catalog.h5"

(S108)